docs/ref/contrib/postgres/fields.txt

```diff
@@ -357,6 +357,23 @@ Returns objects where the given key is in the data. Uses the SQL operator
     >>> Dog.objects.filter(data__has_key='owner')
     [<Dog: Meg>]
 
+.. fieldlookup:: hstorefield.has_any_keys
```

Comment:
when squashing would be nice to leave this feature and its tests/docs in a separate commit